neo/SmartContract/StateMachine.py

```diff
@@ -85,6 +83,42 @@ def CheckStorageContext(self, context):
 
         return False
 
+    def ExecutionCompleted(self, engine, success, error=None):
+
+        # commit storages right away
+        if success:
+            self.Commit()
+
+        height = Blockchain.Default().Height
+        tx_hash = engine.ScriptContainer.Hash
+
+        # dispatch all notify events, along with the success of the contract execution
+        for notify_event_args in self.notifications:
+
+            dispatch_smart_contract_event(SmartContractEvent.RUNTIME_NOTIFY, notify_event_args.State, notify_event_args.ScriptHash.ToString(), height, tx_hash.ToString(), success)
```

Comment:
What data type is `notify_event_args.State` here? Would be great for usability if Runtime.Notify events always have a list of python objects as payload.